contracts/cw-core/examples/schema.rs

```diff
@@ -3,8 +3,10 @@ use std::fs::create_dir_all;
 
 use cosmwasm_schema::{export_schema, remove_schemas, schema_for};
 
-use cw_governance::msg::{ExecuteMsg, InstantiateMsg, QueryMsg};
-use cw_governance::query::DumpStateResponse;
+use cw_core::{
```

Comment:
`cw_core` feels like it is a package that is part of cosmwasm std or smth.
Maybe best to rename to `cw_dao_core` or `cw_da0_core`

Reply:
> `cw_core` feels like it is a package that is part of cosmwasm std or smth. Maybe best to rename to `cw_dao_core` or `cw_da0_core`

Completely agree with this.

Reply:
+1 cw_dao_core

Reply:
Still want this to happen... will make a ticket.

Reply:
https://github.com/DA0-DA0/dao-contracts/issues/255